04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -1347,6 +1391,7 @@
        "{'mydb_user': 'datalab',\n",
        " 'description': 'db01',\n",
        " 'tempfilePath': '/net/dl1/temp',\n",
+       " 'mydb_password': 'datalab',\n",
```

Comment:
So, this HAD been fixed before. TWICE if I'm not mistaken. Why is the mydb password showing up again?? I'm pinging Mike, Adam, and Wendy.

Reply:
Looks like it's been fixed, I ran it again and this time the mydb password does not show up.